templates/swiper-gallery-preview.html.twig

```diff
@@ -1,30 +1,27 @@
-{% if preview_type == 'thumbs' and footer_thumbs is not empty %}
-    <article class="media media-image">
-        <section>{{ footer_image }}</section>
-    </article>
+<article class="gallery__preview-image">
+  {{ media }}
```

Comment:
the media is not for the thumbs preview, it could contain copyright and social media icons (e.g. Galerie-Aufmacher in haus.de), instead the footer_image should be used.